getting_started/step_by_step/singletons_autoload.rst

```diff
@@ -224,11 +224,10 @@ current scene and replace it with the requested one.
         GetTree().SetCurrentScene(CurrentScene);
     }
 
-As mentioned in the comments above, we need to avoid the situation of deleting
-the current scene while it is still being used (i.e. its code is still running),
-so using :ref:`Object.call_deferred() <class_Object_method_call_deferred>`
-is required at this point. The result is that the second function will run
-at a later time when any code from the current scene has completed.
+Using :ref:`Object.call_deferred() <class_Object_method_call_deferred>`,
+the second function will only run once all code from the current scene has
+completed. Do not delete the current scene while it is still being used
```

Comment:
Not deleting the scene while it's used is the purpose of `call_deferred`, so the sentence should start with e.g. "This prevents deleting the current while it is still being used".

Reply:
How does this new edit sound?

Reply:
Sounds great, thanks.